app/graphql/users/mutations.js

```diff
@@ -1,10 +1,18 @@
 const { gql } = require('apollo-server'),
   { userLoggedIn } = require('../events'),
-  userService = require('../../services/users');
+  { validateUser } = require('../../schemas/users'),
+  User = require('../../models').user;
 
 module.exports = {
   mutations: {
-    createUser: (_, params) => userService.create(params),
+    createUser: async (_, user) => {
+      try {
+        await validateUser(user);
+        return User.createUser(user);
+      } catch (err) {
+        throw err;
```

Comment:
just this?

Reply:
in validateUser and createUser function I'm handle errors, so In this point the error has format. For this reason I wrote directly throw errors. Moreover, Add logger description in both cases.